### DISCUSSION TITLE
[Research Spike] Email server

### DISCUSSION THREAD

**Discussion by cedricschwyter** [lumium-technologies/lumium#213] [Ideas]
we need to be able to send emails from our backend to customers for various reasons (email verification, notifications, password reset etc). we need to decide on how to implement this on the backend

**Comment by ReddixT** [comment]
might need to setup an own mail server or get a provider which does it for us

**Reply by cedricschwyter** [reply]
i'm not too familiar, but i think an own server poses a problem in that you are rate limited in how many emails you can send... we should look for a provider i guess
